compiler-plugin/src/main/scala/scalapb/compiler/GeneratorParams.scala

```diff
@@ -49,6 +50,8 @@ object GeneratorParams {
             case "no_lenses" => Right((params.copy(lenses = false), unrecognized))
             case "retain_source_code_info" =>
               Right((params.copy(retainSourceCodeInfo = true), unrecognized))
+            case "scala3Sources" =>
```

Comment:
For consistency with the other flags, it should be "scala3_sources".

Reply:
good point.